components/compliance-service/api/reporting/server/server.go

```diff
@@ -153,36 +152,25 @@ func (srv *Server) Export(in *reporting.Query, stream reporting.ReportingService
 		return err
 	}
 
-	err = exportReports(formattedFilters, srv.es, exporter)
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
-func exportReports(filters map[string][]string, esr *relaxting.ES2Backend, sendResult exportHandler) error {
```

Comment:
This function was only called by the one directly above it, and reading this code it seemed unnecessary and caused some extra mental translation between the variables.